common/src/main/scala/org/apache/comet/vector/NativeUtil.scala

```diff
@@ -205,7 +205,7 @@ class NativeUtil {
       case v @ (_: BitVector | _: TinyIntVector | _: SmallIntVector | _: IntVector |
           _: BigIntVector | _: Float4Vector | _: Float8Vector | _: VarCharVector |
           _: DecimalVector | _: DateDayVector | _: TimeStampMicroTZVector | _: VarBinaryVector |
-          _: FixedSizeBinaryVector | _: TimeStampMicroVector) =>
+          _: FixedSizeBinaryVector | _: TimeStampMicroVector | _: UInt8Vector) =>
```

Comment:
Why we need to handle `UInt8Vector`?

Reply:
Is it used in `Covariance`/`CovariancePop` state types?

Reply:
I think it's for state field [`count`](https://github.com/apache/arrow-datafusion/blob/main/datafusion/physical-expr/src/aggregate/covariance.rs#L167)

Reply:
I think it is `UInt64`? But what you add is `UInt8Vector`?

Reply:
Seems `UInt8Vector` is for `UInt64`. The name is confusing. https://github.com/apache/arrow/blob/main/java/vector/src/main/java/org/apache/arrow/vector/UInt8Vector.java#L37

Reply:
Oh...the naming of Java Arrow API...